### DISCUSSION TITLE
Is it possible to make union with primitives?

### DISCUSSION THREAD

**Discussion by rostyslav-melnychuk-official** [MichalLytek/type-graphql#1413] [Q&A] [answered]
Hello!

My data model has next prop:
```roll!: number[] | number[][]```
What is a proper way to describe it?

I've tried next: 
```@Field(() => [Number | [Number]])```

and

```
const RollUnion = createUnionType({
  name: 'RollUnion', 
  types: [Number, Number[]] as const,
})

  @Field(() => [RollUnion])
  roll!: number[] | number[][]
  ```
Nothing works

What am I doing wrong?

**Comment by MichalLytek** [comment, marked as answer]
It's not permitted by GraphQL spec.
